### DISCUSSION TITLE
Pros/Cons for 125mhz and 13.65 khz Reader

### DISCUSSION THREAD

**Discussion by Mannshoch** [MiczFlor/RPi-Jukebox-RFID#2058] [Q&A]
I'm planing to make a Phoniebox

I have a Question for the RFID Reader and Tags.

- What are the Pros/Cons for 125mhz and 13.65 khz Reader.
- Currently, I'm interested in this 13.65khz tag to stick it on toys or sew it into: https://www.amazon.de/NFC-Tag-NFC-Tag-Aufkleber-Speicher-kompatibel-NFC-Telefonen/dp/B07Q43PXMR
- How do you design the credit-card tags? Could I print on them somehow or do you use sticking paper?

**Comment by Mannshoch** [comment]
Could I use this scanner together with this chips?
scaner: https://www.amazon.de/dp/B082F8G4T2
chips: https://www.amazon.de/dp/B07Q43PXMR

Is it a problem, that the chips are writable?
Do I need to write the data for using?

**Comment by Mannshoch** [comment]
My Phonie Box with this components and 13.56Mhz works. Even My Mobilephone and normal CreditCards:

Scanner https://www.amazon.de/dp/B082F8G4T2
Chips https://www.amazon.de/dp/B07Q43PXMR
Karten: https://www.amazon.de/dp/B07Q6BYVK3
Speaker: https://www.amazon.de/gp/product/B00JRW0M32
Bord: https://www.amazon.de/dp/B01CD5VC92
Entstörer; https://www.amazon.de/dp/B0171PQLB8
Powerbank: https://www.amazon.de/gp/product/B08XQ43HZT

Only one Problem, the mode for run as long as a chip is scanned do not work that way. No play is longer than some seconds.

**Reply by AlvinSchiller** [reply]
Hi, great that its working so far :)

> Only one Problem, the mode for run as long as a chip is scanned do not work that way. No play is longer than some seconds.

Sounds similar to https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/1991

**Reply by AlvinSchiller** [reply]
Did you solve your problem?

**Reply by Mannshoch** [reply]
No, I didn't use the fix mentioned in #1991 because I wait for a regular update.

**Reply by AlvinSchiller** [reply]
ok, but unfortunatly there will be no fix for this currently.
Its wasn't yet reproducable and too few information available. 

Maybe you can help out and report your setup in https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/1991?

**Reply by Mannshoch** [reply]
I didn't modify anything. So, I assume it's a standard installation. The only difference is here that I use 13.56khz and not 125mhz

**Reply by AlvinSchiller** [reply]
I meant what Raspberry Pi Model, which OS Version (buster, bullseye - 32/64 bit) and installation path (with or without Spotify) do you use.

If possible also supply your PhonieboxInstall.conf (clear out passwords) and after the scan the output of "tail -n 200 /var/log/syslog".
Please post this in https://github.com/MiczFlor/RPi-Jukebox-RFID/issues/1991
